ssl/tls13_enc.c

```diff
@@ -581,9 +581,18 @@ int tls13_change_cipher_state(SSL *s, int which)
         goto err;
     }
 
-    if (label == server_application_traffic)
+    if (label == server_application_traffic) {
         memcpy(s->server_app_traffic_secret, secret, hashlen);
-    else if (label == client_application_traffic)
+        /* Now we create the exporter master secret */
+        if (!tls13_hkdf_expand(s, ssl_handshake_md(s), insecret,
```

Comment:
Shouldn't we remove the other generation of this a bit above here?

Reply:
Yes, it was removed in be25f3bd9b6534ab4c80d9c07b2beeb7eafe7b0e